docs/source/examples/managed-jobs.rst

```diff
@@ -245,27 +275,6 @@ Real-World Examples
 * PyTorch Lightning DDP, CIFAR-10: `YAML <https://github.com/skypilot-org/skypilot/blob/master/examples/spot/lightning_cifar10.yaml>`__
 
 
-.. _on-demand:
-
-Managed On-Demand/Reserved Jobs
```

Comment:
Maybe let's keep this in.

Reply:
I checked, don't see any. There was a reference at the top of the page but it's also removed in this PR.